package.json

```diff
@@ -32,6 +32,7 @@
     "seed": "ts-node src/seed.ts"
   },
   "dependencies": {
+    "@css-inline/css-inline": "^0.14.1",
```

Comment:
why do you have this package here? what does it do

Reply:
i had an error in one of my git pull from the main branch. the server was throwing an error for that package.